package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "scripts": {
     "dev": "nf start",
+    "dev:oidc": "VUE_APP_OIDC_ENABLED=true npm run dev",
```

Comment:
Does this also need to start nginx? As-is, I feel like this would only work if you've already run `npm run dev`. Would it work to run `nf start` here?

Reply:
Ooh never mind, it calls `npm run dev`, which starts nginx. 🤦